cmd/bnsd/app/lateinit.go

```diff
@@ -0,0 +1,22 @@
+package bnsd
+
+/*
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/x/lateinit"
+	"github.com/iov-one/weave/x/msgfee"
+)
+
+func init() {
+	// cond:gov/rule/0000000000000003 is owned by technical executors
+	lateinit.MustRegisterGConfig(
+		"msgfee",
```

Comment:
Is this this init ID? Can we make it a little more self-explanary, like "set_msgfee_admin_to_economic_executors"

Reply:
That is the package name that the initialization will be run on.
```go
func MustRegisterGConfig(
	pkgName string,
	requiredSigner weave.Address,
	entity orm.Model,
) {
	MustRegister(
		pkgName+"/configuration",
		requiredSigner,
		[]byte(pkgName),
		gconf.NewConfigurationModelBucket(),
		entity,
	)
}
```
`MustRegister` merges `pkgName` and `/configuration` string, which becomes `msg_fee/configuration` in this instance. Then it is fed to MustRegister as `initID`:
```go
// MustRegister registers an initialization function for given entity.
func MustRegister(
	initID string,
	requiredSigner weave.Address,
	entityID []byte,
	bucket orm.ModelBucket,
	entity orm.Model,
)
```